README.md

```diff
@@ -13,11 +13,11 @@ rill start my-rill-project
 ```
 
 <h3 align="center">
-  <a href="https://docs.rilldata.com/home/get-started">Quickstart</a>
+  <a href="https://docs.rilldata.com/get-started/get-started">Quickstart</a>
   <span> · </span>
-  <a href="https://docs.rilldata.com/home/install">Install</a>
+  <a href="https://docs.rilldata.com/get-started/install">Install</a>
   <span> · </span>
-  <a href="https://docs.rilldata.com/home/get-started#example-projects">Example Projects</a>
+  <a href="https://docs.rilldata.com/get-started/get-started#example-projects">Example Projects</a>
```

Comment:
This `/get-started/get-started` link should also be changed accordingly

Reply:
I searched `/get-started/get-started` and see a couple more cases:
- `WelcomeMessage.svelte`
- `web-admin/src/routes/[organization]/+page.svelte`